### PR TITLE
add Okx futures 

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/837

### CODE REVIEW

File: Trading/Exchange/okx/okx_exchange.py [hunk 1]
```diff
@@ -14,50 +14,49 @@
...
+import octobot_trading.exchanges.connectors.ccxt.enums as ccxt_enums
+import octobot_trading.personal_data as trading_personal_data
 
 
 class Okx(exchanges.RestExchange):
     MAX_PAGINATION_LIMIT: int = 100  # value from https://www.okex.com/docs/en/#spot-orders_pending
     DESCRIPTION = ""
-
-    # FROM https://www.okex.com/docs-v5/en/#overview-demo-trading-services
-    SANDBOX_MODE_HEADERS = {"x-simulated-trading": "1"}
```
Comment (Herklos) [inline Trading/Exchange/okx/okx_exchange.py:27]: Is sandbox supported by ccxt now?

File: Trading/Exchange/okx/okx_exchange.py [hunk 2]
```diff
@@ -14,50 +14,49 @@
...
-
-    def __init__(self, config, exchange_manager):
-        super().__init__(config, exchange_manager)
-
-        if self.exchange_manager.is_sandboxed:
-            self.connector.add_headers(self.SANDBOX_MODE_HEADERS)
+    REQUIRES_MOCKED_EMPTY_POSITION = True   # https://www.okx.com/learn/complete-guide-to-okex-api-v5-upgrade#h-rest-2
```
Comment (Herklos) [inline Trading/Exchange/okx/okx_exchange.py:31]: 👍

File: Trading/Exchange/okx/okx_exchange.py (hunk 1 above)
Comment (Herklos) [reply Trading/Exchange/okx/okx_exchange.py:27]: 👍

**Review by Herklos — APPROVED** [review]
💯

**Comment by GuillaumeDSM** [comment]
merged with https://github.com/Drakkar-Software/OctoBot-Tentacles/pull/867

**Pull request closed without merging**